src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/BlobDispatcher.java

```diff
@@ -39,4 +40,26 @@ public void dispatch(
             targetStorageAccount
         );
     }
+
+    public void moveBlob(
+        BlobClient blob,
+        String destinationContainer,
+        TargetStorageAccount targetStorageAccount
+    ) {
+        logger.info(
+            "Move file from: {} to Container: {}. Storage: {}",
+            blob.getBlobUrl(),
+            destinationContainer,
+            targetStorageAccount
+        );
+
+        blobContainerClientProxy.moveBlob(blob, destinationContainer, targetStorageAccount);
```

Comment:
Or maybe check the status of PollResponse here, return the result in such a case.

Reply:
I think we do not need dispatcher . I will tidy up the code, first need to find the way to upload for PCQ and crime. 
after that CFT file copy need to change as well as we do not need to transfer all zip file.